notebooks/train_with_additional_metadata/benchmarking/comparison_rmse_plot.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
Everything in notebooks, does not have to be added in this PR. Still useful locally, but I will remove it from this PR